docs/source/en/tasks/image_classification.mdx

```diff
@@ -213,7 +213,7 @@ At this point, only three steps remain:
 ...     data_collator=data_collator,
 ...     train_dataset=food["train"],
 ...     eval_dataset=food["test"],
-...     tokenizer=feature_extractor,
+...     tokenizer=image_processor,
```

Comment:
I don't mean to change this in current PR - just feel mind blowing when I see this line 😲 😲 😲 😲

Reply:
I know 👀  - I found this pretty confusing tbh when I first saw it 🙈